Test/Mono.Cecil.Tests.csproj

```diff
@@ -1,31 +1,30 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectGuid>{A47B1F49-A81A-43E8-BE6B-DD28AF2C4055}</ProjectGuid>
-    <RootNamespace>Mono.Cecil.Tests</RootNamespace>
-    <AssemblyName>Mono.Cecil.Tests</AssemblyName>
+    <TargetFrameworks>net40</TargetFrameworks>
```

Comment:
if u add multiple frameworks u can run the tests against each of those

Reply:
Yes, but for now I kept it the way it was being used in the past.